test/spec/controllers/challengesCtrlSpec.js

```diff
@@ -174,4 +174,33 @@ describe('Challenges Controller', function() {
       expect(scope.filterChallenges(notOwnNotMem)).to.eql(true);
     });
   });
+
+  it('Clones a challenge', function() {
+    var copyChallenge = new challenges.Challenge({
+      name: 'copyChallenge',
+      description: 'copyChallenge',
+      habits: [],
+      dailys: [],
+      todos: [],
+      rewards: [],
+      leader: user._id,
+      group: "copyGroup",
+      timestamp: +(new Date),
+      members: [user],
+      official: false,
+      _isMember: true
+    });
+    scope.clone(copyChallenge)
+
+    expect( scope.obj.name ).to.eql(copyChallenge.name );
+    expect( scope.obj.description ).to.eql(copyChallenge.description );
+    expect( scope.obj.habits ).to.eql(copyChallenge.habits );
```

Comment:
does using `scope.newChallenge` work here? I think that'd be more descriptive than `scope.obj`